kiran.bash

```diff
@@ -0,0 +1,7 @@
+mkdir code
```

Comment:
Script files usually should have descriptive names, so for example `setupProject.sh`. Also, you should always add some comments at the top of each script saying what it is for and who created it. Additionally, the idea was for the script to create the `.gitkeep` files and put them in the correct directories. You also could add some `echo` statements to let the user know what is happening.